charts/drupal/templates/_helpers.tpl

```diff
@@ -373,23 +373,21 @@ done
 {{- define "drupal.extract-reference-data" -}}
 set -e
 if [[ "$(drush status --fields=bootstrap)" = *'Successful'* ]] ; then
-  # Figure out which tables to skip.
-  IGNORE_TABLES=""
-  IGNORED_TABLES=""
-  for TABLE in `drush sql-query "show tables;" | grep -E '{{ .Values.referenceData.ignoreTableContent }}'` ;
+  echo "Dump reference database."
+  mkdir /tmp/reference-data/
+  for table in $(drush sql-query 'show tables;');
   do
-    IGNORE_TABLES="$IGNORE_TABLES --ignore-table=$DB_NAME.$TABLE";
-    IGNORED_TABLES="$IGNORED_TABLES $TABLE";
+    # Add --no-data parameter for tables that match .Values.referenceData.ignoreTableContent.
+    [[ "${table}" =~ {{ .Values.referenceData.ignoreTableContent }} ]] && nodata='--no-data' || nodata=''
+    echo "Dumping table: ${table}"
+    mysqldump --user="${DB_USER}" --password="${DB_PASS}" --host="${DB_HOST}" "${nodata}" "${DB_NAME}" "${table}" > /tmp/reference-data/"${table}".sql
```

Comment:
i see a problem here that the dumps may become inconsistent.
cant lock db during individual table dumping process.
even if this works initially, there will be case where people become confused due to strange problems they witness.
it will be quite hard to debug. it would take silta support resources, and generate dissatisfaction.
there should be some kind of a workaround.

i still think that the general idea is good - parallel import.

we may have a zoom session about options how to do it safely/consistently/atomically.

I'm currently putting `request changes` for this particular item.
to make sure we wont backstab ourselves.